packages/desktop/components/menus/CollectibleDetailsMenu.svelte

```diff
@@ -12,28 +12,11 @@
     export let menu: Menu = undefined
     export let nft: INft
 
-    $: url = nft?.metadata?.uri && composeUrl(nft.metadata.uri)
+    $: url = nft?.composedUrl ?? composeUrlFromNftUri(nft?.metadata?.uri)
     $: isLocked = isNftLocked(nft)
+    $: isBurnDisabled = isLocked || !isIrc27Nft(nft)
```

Comment:
We could also implement NFT burning functionality by sending to the 0x0 address (null address) or `0x000000000000000000000000000000000000dEaD` (burn address).

Reply:
Ah true, should be easy to do once we get send flow impl. working